app/services/jobs/return-logs/process-return-logs.service.js

```diff
@@ -5,19 +5,22 @@
  * @module CreateReturnLogsService
  */
 
-const FetchReturnLogsService = require('./fetch-return-logs.service.js')
+const { calculateAndLogTimeTaken, currentTimeInNanoseconds } = require('../../../lib/general.lib.js')
 const CreateReturnLogsService = require('./create-return-logs.service.js')
+const FetchReturnLogsService = require('./fetch-return-logs.service.js')
 
 /**
  * Creates the return logs for the next cycle
  */
 async function go (isSummer, licenceReference) {
   try {
+    const startTime = currentTimeInNanoseconds()
     const returnLogs = await FetchReturnLogsService.go(isSummer, licenceReference)
 
     if (returnLogs.length > 0) {
       await CreateReturnLogsService.go(returnLogs)
     }
+    calculateAndLogTimeTaken(startTime, 'Process return logs complete', { isSummer, licenceReference })
   } catch (error) {
     global.GlobalNotifier.omfg('Create return logs job failed', null, error)
```

Comment:
Ideally, the log messages should be the same. Using the pattern in other services.

```javascript
    calculateAndLogTimeTaken(startTime, 'Create return logs job complete', { isSummer, licenceReference })
  } catch (error) {
    global.GlobalNotifier.omfg('Create return logs job failed', { isSummer, licenceReference }, error)
```